ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -923,3 +923,28 @@ def test_jax_numpy_compress(
         a=arr[0],
         axis=ax,
     )
+    
+    
+@handle_frontend_test(
+    fn_tree="jax.numpy.iterable",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+)
+def test_jax_numpy_iterable(
+    dtype_and_x,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
```

Comment:
This is supposed to be `y` as in the function signature
```suggestion
        y=x[0],
```